stac_fastapi/sfeos_helpers/stac_fastapi/sfeos_helpers/database/index.py

```diff
@@ -120,11 +176,11 @@ async def delete_item_index_shared(settings: Any, collection_id: str) -> None:
     client = settings.create_client
 
     name = index_alias_by_collection_id(collection_id)
-    resolved = await client.indices.resolve_index(name=name)
+    resolved = await client.indices.resolve_index(name=name, ignore=[404])
```

Comment:
What does the `ignore` parameter do? I cannot find it in the ES/OS docs: 

- https://elasticsearch-py.readthedocs.io/en/latest/api/indices.html#elasticsearch.client.IndicesClient.resolve_index
- https://opensearch-project.github.io/opensearch-py/api-ref/clients/indices_client.html#opensearchpy.client.indices.IndicesClient.resolve_index